english/meditate/meditate.md

```diff
@@ -20,6 +22,8 @@ Answer: sit, stare, fill your consciousness with Git
 
 Afterwards, you decide to:
 
+[attend a warriors game](../warriors-game/warriors-game.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.